elastic/kibana
# [Index Management] Add enrich policies permission check

Closes: https://github.com/elastic/kibana/issues/167826

## Summary
This PR adds a permissions check based on the `manage_enrich` privilege before allowing the user to navigate to the enrich policies tab or the create enrich policy page.

##### How to test
1. Start elasticsearch with `yarn es snapshot --license=trial -E path.repo=/tmp/snapshots` and kibana with `yarn start`
2. Create a new user that doenst have `manage_enrich` privileges
3. Navigate to index management and click on enrich policies tab. 
4. Verify that:
  * A callout saying you dont have permissions shows up


<details>
<summary>Screenshots</summary>

![Screenshot 2023-10-27 at 09 13 49](https://github.com/elastic/kibana/assets/1191206/b6786191-6f92-40b1-97f7-210b3ac6b176)

</details>
